verifying/verifying.go

```diff
@@ -2,25 +2,17 @@ package verifying
 
 import (
 	"bytes"
+	"crypto/aes"
+	"crypto/cipher"
 	"errors"
 	"fmt"
 	"math/big"
+	"unsafe"
 
-	"github.com/spacemeshos/post/config"
 	"github.com/spacemeshos/post/oracle"
 	"github.com/spacemeshos/post/shared"
 )
 
-type (
-	Config = config.Config
-)
-
-var (
-	WorkOracle = oracle.WorkOracle
-	FastOracle = oracle.FastOracle
-	UInt64LE   = shared.UInt64LE
-)
```

Comment:
I'm curious - why did you remove the re-exports everywhere?

Reply:
I removed all from the last implementation. I don't quite see the benefit of this? What is the advantage of writing:

```go
res, err := WorkOracle(
    oracle.WithId(...),
    oracle.WithPosition(...),
    ...
)
```

over

```
res, err := oracle.WorkOracle(
    oracle.WithId(...),
    oracle.WithPosition(...),
    ...
)

?

If one calls `WorkOracle` they always have to import `github.com/spacemeshos/post/oracle` anyway, but have 2 ways of calling the same function. `FastOracle` doesn't exist any more and `UInt64LE` isn't used any more.